Remotion/ObjectBinding/Web/Globalization/BocList.fr.resx

```diff
@@ -174,13 +174,13 @@
   <data name="Remotion.ObjectBinding.Web.UI.Controls.BocList.ValidationFailuresFoundInOtherListPagesErrorMessage" xml:space="preserve">
     <value>Des erreurs sont survenues sur au moins une autre page.</value>
```

Comment:
please check the wording for FR and IT regarding error/invalid input. It seems like the correct wording is "Saisie non valable" / "Input non valido" (correctly adjusted for grammar). Doku should do a final check here before we're done.